rc/gdscript.kak

```diff
@@ -0,0 +1,110 @@
+
+    join() { sep=$2; eval set -- $1; IFS="$sep"; echo "$*"; }
+
+    # Add the language's grammar to the static completion list
+    printf %s\\n "declare-option str-list gdscript_static_words $(join "${keywords} ${values} ${types} ${builtin_classes}" ' ')"
+
+    printf %s "
+        add-highlighter shared/gdscript/code/ regex '\b($(join "${keywords}" '|'))\b' 0:keyword
+        add-highlighter shared/gdscript/code/ regex '\b($(join "${values}" '|'))\b' 0:value
+        add-highlighter shared/gdscript/code/ regex '\b($(join "${types}" '|'))\b' 0:type
+        add-highlighter shared/gdscript/code/ regex '\b($(join "${builtin_classes}" '|'))\b' 0:type
```

Comment:
This probably won't match the `@GDScript` and `@GlobalScope` symbols, because they start with a `@` and the regex requires `\b`.